apps/drupal/patternlab.theme

```diff
@@ -39,6 +39,29 @@ function patternlab_preprocess_block(&$variables) {
     } else {
       $variables['logo_img_src'] = $logo_url;
     }
+  }
+
+  if ($id === 'system_powered_by_block') {
+    $variables['content']['#markup'] = '<p>&copy; Phase2 ' . date('Y') . ' </p>';
+  }
+
+}
 
+/**
+ * Implements hook_preprocess_region().
+ */
+function patternlab_preprocess_region__header(&$variables) {
+  $blocks = [
+    'branding' => 'patternlab_sitebranding',
+  ];
+  foreach ($blocks as $key => $value) {
+    // Begins with value.
```

Comment:
I understand it begins with value, but I'm really not clear what the goal is of this code snippet. Stronger what & why comment?

Reply:
I'll expand this out.